korge-foundation/src/korlibs/memory/ByteArrayExtention.kt

```diff
@@ -0,0 +1,17 @@
+package korlibs.memory
+
+import korlibs.number.Half
+
+private fun ByteArray.u8(offset: Int): Int = this[offset].toInt() and 0xFF
+private inline fun ByteArray.get16LE(offset: Int): Int = (u8(offset + 0) shl 0) or (u8(offset + 1) shl 8)
+private inline fun ByteArray.get16BE(offset: Int): Int = (u8(offset + 1) shl 0) or (u8(offset + 0) shl 8)
```

Comment:
Aren't these exposed publicly already in other package?

Reply:
I dont know about that. They were private in the original class